lib/toml-parser.js

```diff
@@ -172,16 +172,7 @@ function isList (obj) {
   return obj[_type] === LIST
 }
 
-// in an eval, to let bundlers not slurp in a util proxy
-let _custom
-try {
-  const utilInspect = eval("require('util').inspect")
-  _custom = utilInspect.custom
-} catch (_) {
-  /* eval require not available in transpiled bundle */
-}
-/* istanbul ignore next */
-const _inspect = _custom || 'inspect'
+const _inspect = 'inspect'
```

Comment:
https://nodejs.org/api/util.html#util_util_inspect_custom

```suggestion
const _inspect = Symbol.for('nodejs.util.inspect.custom')
```